tests/qiita-sql/91.sql

```diff
@@ -14,6 +14,19 @@ INSERT INTO
         study_abstract
     )
 VALUES
+    (
```

Comment:
Amplicon actually inserts preps to study 1 (exists as default in qiita) and 2, which doesn't exist.